src/main/java/Main.java

```diff
@@ -53,8 +56,11 @@ public static void main(String[] args) {
         NewUserView newUserView = new NewUserView(new NewUserController(new NewUserInteractor(new NewUserPresenter(newUserViewModel, adminViewModel), new UserDbGateway())));
         newUserViewModel.addObserver(newUserView);
 
+        NewFacilityView newFacilityView = new NewFacilityView(new NewFacilityController(new NewFacilityInteractor(new NewFacilityPresenter(newFacilityViewModel), new FacilityDbGateway())));
+        newFacilityViewModel.addObserver(newFacilityView);
+
         UserDbGateway userDb = new UserDbGateway();
-        //userDb.fileReset();
+        userDb.fileReset();
```

Comment:
For now, lets leave it in because we can alter the users and stuff while testing. The problem becomes that we don't have the new facility and new product use cases so we cannot do this within the GUI. For final this will need to be removed though. All of this code at the bottom of main is just for testing purposes at the moment. 

We should add a logout button regardless, but this still wont help that we need to manually add users and products to the db because that isn't implemented yet.